README.md

```diff
@@ -7,7 +7,61 @@ LeBotTel is an easy to use python package to create your own Telegram bots for d
 The name LeBotTel is combined from "LeCun", "RoBot" and "Telegram". The "Le" in the name comes from Yann LeCun, to honor his work on neural networks (has nothing to do with this project, but he is a great scientist and pioneer in the field of neural networks (LeNet)). What has this bot to do with neural networks? We want to use this bot to log the training of neural networks. "Bot" comes from RoBot, because this is what the goal of this project is. And "Tel" comes from Telegram, because its a bot for Telegram. The "Pi" in our logo represents looks similar to the "tT" in "LeBotTel" and stands for the used Programming Language Python.
 
 ## Quick Start
-TODO add quick start / API documentation / how to install / ...
+
+#### Installation
+
+```bash
+pip install LeBotTel
+```
+
+#### Send a message
+
```

Comment:
maybe add a section on how to get the token and chatid before this. I think there is a bug regarding the chatid at the moment so we can leave this for now